css/properties/align-items.json

```diff
@@ -2,306 +2,372 @@
-                  }
-                ]
-              }
-            ],
-            "firefox_android": [
-              {
-                "version_added": "20",
-                "notes": "Multi-line flexbox has been supported since Firefox 28."
-              },
-              {
-                "prefix": "-webkit-",
-                "version_added": "49"
-              },
-              {
-                "version_removed": true,
-                "version_added": "18",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.flexbox.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              },
-              {
-                "prefix": "-webkit-",
-                "version_added": "48",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.prefixes.webkit",
-                    "value_to_set": "true"
-                  }
-                ]
-              }
-            ],
-            "ie": {
-              "version_added": "11",
-              "notes": "In Internet Explorer 10 and 11, if column flex items have <code>align-items: center;</code> set on them and their content is too large, then they will overflow the bounds of their container. See <a href='https://github.com/philipwalton/flexbugs#2-column-flex-items-set-to-align-itemscenter-overflow-their-container'>Flexbug #2</a>."
-            },
-            "opera": {
-              "version_added": "12.1"
-            },
-            "opera_android": {
-              "version_added": "12.1"
-            },
-            "safari": {
-              "prefix": "-webkit-",
-              "version_added": "7"
-            },
-            "safari_ios": {
-              "prefix": "-webkit-",
-              "version_added": "7"
-            },
-            "samsunginternet_android": {
-              "version_added": null
-            },
-            "webview_android": [
-              {
-                "version_added": "52"
-              },
-              {
-                "version_added": "29",
-                "partial_implementation": true,
-                "notes": "Older versions of the specification treat absolute positioned children as though they are a 0 by 0 flex item. Later specification versions take the children out of the flow and set their positions based on align and justify properties. Chrome implements the new behavior beginning with Chrome 52."
-              },
-              {
-                "prefix": "-webkit-",
-                "version_added": "21"
-              }
-            ]
-          },
-          "status": {
-            "experimental": false,
-            "standard_track": true,
-            "deprecated": false
-          }
-        },
-        "first_last_baseline": {
+        "flex_context": {
+          "description": "Supported in Flex Layout",
```

Comment:
This should be a sub-item of `__compat`.

Reply:
I'm not sure I understand this comment - I was going on a suggestion by @wbamberg which had this outside __compat, so perhaps he could comment to clarify.

Reply:
It's my mistake: yes, `description` should be inside `__compat` :).